README.adoc

```diff
@@ -262,6 +262,27 @@ Moreover, you can also use link:https://git-scm.com/book/en/v2/Customizing-Git-G
 * Check that the version of fmt-maven-plugin and other integrations remains the same!
 ====
 
+== Convention for unit tests
```

Comment:
Je n’incluerais pas ceci, cela duplique essentiellement les conventions du cours, or (pour le moment au moins) les contributeurs sont ceux qui suivent le cours… OPT.

Reply:
Fix in 6515e0ee72247f99ae2607a0c61d7ce6a3bb6838